components/Date.tsx

```diff
@@ -0,0 +1,76 @@
+import React from "react";
+
+interface DateProps {
+  yearDataList: {
+    year: string;
+    team: { title: string; members: { name: string; cargo: string }[] }[];
+  }[];
```

Comment:
Please, make a interface/type for each subcategory inside `team` key to increase readability of this interface